README.md

```diff
@@ -6,7 +6,7 @@ A free, light and easy to use desktop app to generate PGP key pairs, encrypt and
 
 ## Download it ⬇️
 
-- From the repo's Github page: [releases](https://github.com/aliceandbob-io/aliceandbob-desktop/releases)
+- From the repo's Github page: [releases]( https://www.electronjs.org/apps/aliceandbob-io)

```

Comment:
The 2 existing links should not be deleted. Instead, can you please add the new link after the 2 existing links?
It should read as follows: From the [electronjs.org page]( https://www.electronjs.org/apps/aliceandbob-io)